docs/ldml/tr35-general.md

```diff
@@ -902,157 +902,160 @@ As with other identifiers in CLDR, the American English spelling is used for uni
+
+<a name='unit_constant' href='#unit_constant'>unit_constant</a> 
+<br/>:= [1-9][0-9]* ("e" [1-9][0-9]*)?
+* *Examples:*  
+  * kilowatt-hour-per-100-kilometer
+  * gallon-per-100-mile
+  * per-200-pound
+  * per-12
+* [ wfc:  The numeric value of the unit constant must be an integer greater than one. ]
+* *Notes:*
+    * The normal interpretation of `e` is used, where 2e6 \= 2×10⁶.
+    * The `e` notation is optional: per-100-kilometer and per-1e2-kilometer are equivalent unit\_identifiers.
+    * When constructing identifiers, exponents should be greater than 3 and multiples of 3, even though parsers must accept the wider range.
+  
+<a name='dimensionality_prefix' href='#dimensionality_prefix'>dimensionality_prefix</a> 
+<br/>:= "square-" 
+<br/>   | "cubic-" 
+<br/>   | "pow" ([2-9]|1[0-5]) "-"
+* [ wfc:  Must be value in: \<unitIdComponent type="power"\>. ] 
+* *Notes:*
+    * "pow2-" and "pow3-" canonicalize to "square-" and "cubic-" 
+  
+<a name='simple_unit' href='#simple_unit'>simple_unit</a> 
+<br/>:= (prefix_component "-")* (prefixed_unit 
+<br/>   | base_component) ("-" suffix_component)*
+<br/>   | currency_unit
+<br/>   | ("em" | "g" | "us" | "hg" | "of")
+* *Examples:*
+    * kilometer
+    * meter
+    * cup-metric
+    * fluid-ounce
+    * curr-chf
+    * em
+* *Notes:*
+    * Five simple units are currently allowed as legacy usage, for tokens that wouldn’t otherwise be a base\_component due to length (eg, "g-force").Those are likely to be deprecated in teh future, with conformant aliases added: the "hg" and "of" are already only in deprecated simple\_units.
+  
+<a name='prefixed_unit' href='#prefixed_unit'>prefixed_unit</a>
+    prefix base_component
+* *Examples:*
+    *  kilometer
+
+<a name='prefix' href='#prefix'>prefix</a> 
+<br/>:= si_prefix 
+<br/>   | binary_prefix
+
+<a name='si_prefix' href='#si_prefix'>si_prefix</a> 
+<br/>:= "deka" 
+<br/>   | "hecto" 
+<br/>   | "kilo", …
+* [ wfc:  Must be an attribute value of the `type` in: \<unitPrefix type='…' … power10='…'\> ]
+* *Notes:*
+    * See also [NIST special publication 811](https://www.nist.gov/pml/special-publication-811) 
+
+<a name='binary_prefix' href='#binary_prefix'>binary_prefix</a> 
+<br/>:= "kibi", "mebi", …
+* [ wfc:  Must be an attribute value of the `type` in: \<unitPrefix type='…' … power2='…'\>. ]
+* *Notes:*
+    * See also [Prefixes for binary multiples](https://physics.nist.gov/cuu/Units/binary.html)
+
+<a name='prefix_component' href='#prefix_component'>prefix_component</a> 
+<br/>:= [a-z]{3,∞}
+* [ vc:  must be value in: \<unitIdComponent type="prefix"\>. ]
+* *Notes:*
+    * The set of prefix components often expands in new releases, so the requirement to be one of these attribute values is a validity constraint, not a well-formedness constraint. *
+
+<a name='base_component' href='#base_component'>base_component</a> 
+<br/>:= [a-z]{3,∞}
```

Comment:
infinity symbol? Usually this is `*`? I'm not au courant on your EBNF syntax... (there are instances before/after this one)

Reply:
fixed to follow standard PRE — like {3,}, including other instances.